parlai/agents/bert_ranker/cross_encoder_ranker.py

```diff
@@ -67,6 +67,15 @@ def score_candidates(self, batch, cand_vecs, cand_encs=None):
         nb_cands = cand_vecs.size()[1]
         size_batch = cand_vecs.size()[0]
         text_vec = batch.text_vec
+
+        if self.opt['fix_size'] != -1:
```

Comment:
I'm not sure I quite get this (the patch is clear, the need is not). Shouldn't it be bounded by (text-truncate + label_truncate) already?

Reply:
hmm yes wouldn't `text-truncate` take care of limiting the context length?